spec/models/pacbio/request_library_spec.rb

```diff
@@ -22,7 +22,7 @@
     request_library = create(:pacbio_request_library, tag: create(:tag))
     expect(request_library.tag_oligo).to be_present
     expect(request_library.tag_group_id).to be_present
-    expect(request_library.tag_set_name).to be_present
+    expect(request_library.tag.tag_set.name).to be_present
```

Comment:
Does this want to be `tag_set_name` to test the delegate method? Maybe `expect(request_library.tag_set_name).to eq request_library.tag.tag_set.name`